STYLEGUIDE.md

```diff
@@ -415,14 +415,18 @@ Be sure to use the correct term with components vs functions.
 
 > The `<SignUp />` component accepts the `signUpProps` parameter. The `buildUrlWithAuth()` function accepts the `to` property.
 
-### 4.10 Start and end every Clerk Dashboard operation with specific links to the Clerk Dashboard
+### 4.10 Navigating users should have clear instructions and should include direct links to the Clerk Dashboard
```

Comment:
Reads a little mussy with the gerund at the front like that (am I navigating the user?) How about "Provide users with clear instructions and a direct link when navigating the Clerk Dashboard"?

Reply:
Agreed and updated!